src/lighteval/models/base_model.py

```diff
@@ -485,7 +583,7 @@ def _generate(
             pad_token_id=self.tokenizer.pad_token_id if self.tokenizer.pad_token_id else self.tokenizer.eos_token_id,
             return_dict_in_generate=True,
             output_scores=True,
-            eos_token_id=self.tokenizer.eos_token_id,
+            # eos_token_id=self.tokenizer.eos_token_id,
```

Comment:
I think @lewtun had added this recently to fix a bug on some models, where the eos token was ignored at generation time

Reply:
yeah, but this makes Yi34b-Chat generate until max_tokens and not stop at stop tokens, not sure why still need to investigate that

Reply:
re-added it